llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1067,8 +1070,9 @@ class VPInstructionWithType : public VPInstruction {
 public:
   VPInstructionWithType(unsigned Opcode, ArrayRef<VPValue *> Operands,
                         Type *ResultTy, const VPIRFlags &Flags, DebugLoc DL,
-                        const Twine &Name = "")
-      : VPInstruction(Opcode, Operands, Flags, DL, Name), ResultTy(ResultTy) {}
+                        bool SingleScalar = false, const Twine &Name = "")
```

Comment:
Nit, should this match the VPInstruction arg name?
```suggestion
                        bool IsSingleScalar = false, const Twine &Name = "")
```

Reply:
Updated, thanks